indico/htdocs/sass/modules/_admin.scss

```diff
@@ -36,3 +36,12 @@
         font-weight: bold;
     }
 }
+
+.tasks {
+    a.flower-url {
+        font-weight: bold;
+    }
+    .i-table-widget {
+        width: 815px;
```

Comment:
don't we have an existing and more generic class to get a fixed width like that?